src/providers/info.rs

```diff
@@ -223,18 +232,20 @@ impl<C: Cache> DefaultInfoProvider<'_, C> {
     }
 
     async fn load_chain_info(&self, chain_id: &str) -> ApiResult<Option<ChainInfo>> {
-        let url = format!("{}/v1/chains/{}/", base_config_service_url(), chain_id);
+        let url = format!("{}/v1/chains/{}", base_config_service_url(), chain_id);
```

Comment:
Please note the lack of trailing slash. Currently, this would produce a `404` so we need to decide whether we need it or not.

Reply:
Maybe we should build the url using some URL utils from Rocket

Reply:
I think that if we want to introduce a tool from an external crate here, something from `reqwest` would be more appropriate? https://docs.rs/reqwest/0.4.0/reqwest/struct.Url.html

Reply:
This was fixed as part of https://github.com/gnosis/safe-config-service/pull/110

Now both versions work with a slight difference: ending with slash hits the endpoint directly while without slash redirects to the endpoint with slash